packages/browser-integration-tests/suites/replay/multiple-pages/test.ts-snapshots/seg-5-snap-incremental-chromium

```diff
@@ -5,8 +5,8 @@
       "source": 2,
       "type": 1,
       "id": 12,
-      "x": 41,
-      "y": 90
+      "x": 41.810001373291016,
```

Comment:
are we sure these do not flake 😅

Reply:
Will double check where these are even used, does seem sus